src/main/java/com/esri/samples/localserver/local_server_services/LocalServerServicesController.java

```diff
@@ -50,12 +54,25 @@
 
   private HostServices hostServices;
   private FileChooser packageChooser;
+  private Path appDataPath;
 
   @FXML
   private void initialize() {
 
     if (LocalServer.INSTANCE.checkInstallValid()) {
       LocalServer server = LocalServer.INSTANCE;
+
+      // configure app data path (path length must be short for some services)

```

Comment:
@Rachael-E and I think it might be worth specifying what 'short' means in this context? Character limit, or depth?

Reply:
@JonLavi it looks like this will be covered in the updated doc, so I'd be happy to leave comment as it is.